### DISCUSSION TITLE
Add "howto" link to scan-throttling message

### DISCUSSION THREAD

**Discussion by sirtet** [VREMSoftwareDevelopment/WiFiAnalyzer#496] [Ideas]
![Screenshot_20240914_123931_WiFiAnalyzer](https://github.com/user-attachments/assets/afcffa85-c0f9-4d06-af64-e9ed01f6eb84)

On first startup, a howto was shown on how to deactivate throttling. 
It would be nice if there was a way to open this howto from the place(s) where the warning (see screenshot) is shown

**Comment by VREMSoftwareDevelopment** [comment]
Currently the links to How-To, FAQ and other URLs are available on the Application About page.
